### ISSUE TITLE
Can not run plugin after 0.3.0 upgrade

### ISSUE THREAD

**Issue by Bartuz** [gpbl/webpack-cleanup-plugin#13]
Happened after 0.3.0 upgrade. 

Ran my script `NODE_ENV=production webpack --config build/webpack.production.js --json > stats.json` and got this:

```
/Users/bartuz/work/xxx/frontend/build/webpack.production.js:81
    new WebpackCleanupPlugin({
    ^

TypeError: WebpackCleanupPlugin is not a constructor
    at Object.<anonymous> (/Users/bartuz/work/xxx/frontend/build/webpack.production.js:81:5)
    at Module._compile (module.js:556:32)
    at Object.Module._extensions..js (module.js:565:10)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
    at Module.require (module.js:483:17)
    at require (internal/module.js:20:19)
    at module.exports (/Users/bartuz/work/xxx/frontend/node_modules/webpack/bin/convert-argv.js:80:13)
    at Object.<anonymous> (/Users/bartuz/work/xxx/frontend/node_modules/webpack/bin/webpack.js:39:40)
    at Module._compile (module.js:556:32)
    at Object.Module._extensions..js (module.js:565:10)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
    at Module.runMain (module.js:590:10)
```

Is plugin properly built on npm repo (that's my guess)? `const WebpackCleanupPlugin = require('webpack-cleanup-plugin');` works just fine. Can't track it down

**Comment by gpbl** [comment]
thanks for reporting! checking it out.

**Comment by Bartuz** [comment]
@gpbl Any news on that :) ? I see few other people gave 👍 to this issue.

**Comment by gpbl** [comment]
Thanks @ifamed for the PR! Fix has been published as v0.3.1.
